src/commands/eval/filterFailingTests.ts

```diff
@@ -0,0 +1,26 @@
+import {TestSuite} from "../../types";
+import {readOutput, resultIsForTestCase} from "../../util";
+
+type Tests = NonNullable<TestSuite['tests']>;
+
+export async function filterFailingTests(testSuite: TestSuite, outputPath: string): Promise<Tests> {
+  if (!testSuite.tests) {
+    return [];
+  }
+
+  const {results} = await readOutput(outputPath);
+
+  if (results.version < 2) {
+    throw new Error(`Unsupported output version: ${results.version}`);
```

Comment:
I put this here just cause I'm not familiar with the differences between versions